File: 백준/Bronze/1546. 평균/평균.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().split("\n");

let score = input[1].split(" ").map((i) => +i);
let sum = 0;
const M = Math.max(...score);
const newScore = score.map((val) => (val / M) * 100);
const sumScore = newScore.reduce((acc, cur) => {
  return acc + cur;
});
console.log(sumScore / +input[0]);

File: 백준/Bronze/1546. 평균/평균.py
import sys
input = sys.stdin.readline
N = int(input())
arr = list(map(int,input().split(' ')))
M = max(arr)
new_arr = []
sum = 0
for i in arr:
    new_arr.append(i/M*100)
for i in new_arr:
    sum += i
print(sum/N)